### DISCUSSION TITLE
unlocking piano

### DISCUSSION THREAD

**Discussion by LinaTBell** [Monika-After-Story/MonikaModDev#8719] [QA - tech support + general] [answered]
so... when i checked the amount of time i've spent with monika, it said i've spent 7 days, 12 hours and 49 minutes, which is about 180 hours, and from what i've heard it takes 150 hours to unlock, is there something i'm missing? the affection is at 485, but idk if that's important or not

**Comment by Booplicate** [comment, marked as answer]
The system is dynamic, there isn't an exact number of hours needed to unlock it, 150h is probably too low.
